bin/rust/std/lib.rsspec

```diff
@@ -244,58 +243,59 @@ mod sync {
         struct AtomicUsize;
 
         /*@
-        fix value_inner(v: AtomicUsize) -> usize;
-        lem atomic_to_usize_chunk_vs(p: *AtomicUsize);
+        fix AtomicUsize_inner(v: AtomicUsize) -> usize;
+
```

Comment:
Would it be useful to introduce a helper predicate?
```
pred AtomicUsize(p: *AtomicUsize; v: usize) = *p |-> ?v_ &*& v == AtomicUsize_inner(v_);
```
This would make every assertion that talks about an AtomicUsize much more concise?